src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
```

Comment:
Should be always 4 cards per row
![image](https://user-images.githubusercontent.com/54936044/232194009-abe67c85-7b88-4c42-9775-86daf341b039.png)